### ISSUE TITLE
prestosql/Map.cpp:191, Function:offsetsAligned, Expression: values->size() >= rows.size() (99 vs. 100)

### ISSUE THREAD

**Issue by mbasmanova** [facebookincubator/velox#2984]
```
buck run velox/expression/tests:velox_expression_fuzzer_test -- --velox_fuzzer_enable_complex_types   --velox_fuzzer_max_level_of_nesting 2 --only "element_at,map" --seed 2929397762

I1026 22:51:11.987653 1401984 ExpressionFuzzer.cpp:704] ==============================> Started iteration 0 (seed: 2929397762)
I1026 22:51:11.990541 1401984 ExpressionVerifier.cpp:130] Executing expression: element_at(map("c0",10 elements starting at 270 {[270->76] [76->790] [790->605] null, null, [272->751] [751->98] [98->775] 0, [273->67] [67->403] [403->69] 0, [274->382] [382->795] [795->501] 1, ...}),element_at("c1",2817592878))
I1026 22:51:11.990922 1401984 ExpressionVerifier.cpp:133] 2 vectors as input:
I1026 22:51:11.990978 1401984 ExpressionVerifier.cpp:135]       [DICTIONARY ARRAY<SMALLINT>: 100 elements, 8 nulls], [CONSTANT ARRAY<SMALLINT>: 100 elements, 10 elements starting at 20 {-24223, 24107, null, 10587, 26304, ...}], [ARRAY ARRAY<SMALLINT>: 9 elements, no nulls]
I1026 22:51:11.991128 1401984 ExpressionVerifier.cpp:135]       [DICTIONARY ARRAY<SMALLINT>: 100 elements, 9 nulls], [DICTIONARY ARRAY<SMALLINT>: 100 elements, 12 nulls], [DICTIONARY ARRAY<SMALLINT>: 100 elements, 10 nulls], [DICTIONARY ARRAY<SMALLINT>: 100 elements, 14 nulls], [CONSTANT ARRAY<SMALLINT>: 100 elements, 10 elements starting at 470 {null, null, null, null, null, ...}], [ARRAY ARRAY<SMALLINT>: 72 elements, 8 nulls]
E1026 22:51:11.998283 1401984 Exceptions.h:68] Line: velox/functions/prestosql/Map.cpp:191, Function:offsetsAligned, Expression: values->size() >= rows.size() (99 vs. 100), Source: RUNTIME, ErrorCode: INVALID_STATE
E1026 22:51:11.999745 1401984 ExpressionVerifier.cpp:64] Only simplified path threw exception:
terminate called after throwing an instance of 'facebook::velox::VeloxRuntimeError'
  what():  Exception: VeloxRuntimeError
Error Source: RUNTIME
Error Code: INVALID_STATE
Reason: (99 vs. 100)
Retriable: False
Expression: values->size() >= rows.size()
Function: offsetsAligned
File: velox/functions/prestosql/Map.cpp
Line: 191
```

**Comment by mbasmanova** [comment]
A fix could be to use rows.end() instead of rows.size() in offsetsAligned:

```
    VELOX_CHECK_GE(keys->size(), rows.end());
    VELOX_CHECK_GE(values->size(), rows.end());
```

**Comment by mbasmanova** [comment]
CC: @pedroerp

**Comment by pedroerp** [comment]
I was just looking at this codebase yesterday. I can take this one.

**Comment by pedroerp** [comment]
Fixed in #2997
